README.md

```diff
@@ -18,7 +18,11 @@ We hope you find the resources here helpful in creating and distributing your ow
 
 ## Available Scripts
 
-In the project directory, you can run:
+First install the project depencies with:
+
+### `yarn install`
+
+Then, you can run:
```

Comment:
Then, run `yarn start`. It runs the app in the development mode.\

Reply:
I think i fixed it...